tools/codegen/gen_backend_stubs.py

```diff
@@ -15,11 +15,12 @@
 import tools.codegen.api.dispatcher as dispatcher
 from tools.codegen.api.types import DispatcherSignature
 
-
 # Parses the external backend's yaml, and adds a new BackendIndex for the backend's dispatch key.
 # Returns a Tuple of (backend_key, autograd_key, cpp_namespace, updated BackendIndex mapping)
 ParsedExternalYaml = namedtuple('ParsedExternalYaml', [
-    'backend_key', 'autograd_key', 'cpp_namespace', 'backend_indices'])
+    'backend_key', 'autograd_key', 'cpp_namespace', 'backend_indices', 'codegen'])
```

Comment:
I forget if I already said this - eventually, the external codegen will have more customers than just LTC and XLA, so we probably don't want everyone calling into the same codegen API if it includes some LTC-specific stuff (the codegen'd IR). Maybe there should be a separate `gen_ltc_stubs.py`.

I think it's probably fine to wait until onboarding another backend to think about that though.

Reply:
actually - if you're unconditionally generating IR nodes / TS lowerings in this file, then maybe this should be a separate file. Since pytorch/xla and LTC/TS will want to generate separate things.

Reply:
ok.  no issue making that change.  i guess i'll just do it now, no reason to delay.  And I guess i'll name it `gen_lazy_tensor` since ltc is a little vague and stubs is less relevant with complete codegen involved